modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -397,6 +387,74 @@ private void listenForTableChange() {
 
     /** {@inheritDoc} */
     @Override public Table table(String name) {
```

Comment:
Seems that tables() should be consistent with table().

Reply:
I changed this behavior as it was in javadoc, because the same as Ignite 2.
It returns null if the table does not exist.

Reply:
Changed this behavior.